lib/versioncontrol/github/github_test.go

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package github
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestGithubAPI tests the github releases iterator against the real github
+// api. this test is disabled by default since we use the unauthenticated endpoint
+// which doesn't like frequent request spamming.
+func TestGithubAPI(t *testing.T) {
+	if os.Getenv("TEST_GITHUB_API") != "yes" {
+		t.Skip()
```

Comment:
Can you add a "Github tests disabled" (or whatever words work for you) message to the `Skip()` for visibility?